src/collaborative/ot/ot_specific.ts

```diff
@@ -46,6 +51,11 @@ otRegistry.addTransformation(
   ["FREEZE_COLUMNS", "FREEZE_ROWS"],
   freezeTransformation
 );
+otRegistry.addTransformation(
```

Comment:
add transformation tests please :) :see_no_evil:

Reply:
I already added them in `tests\collaborative\collaborative.test.ts`. Or am I missing some ?

Reply:
Added pure ot tests in `tests\collaborative\ot\ot.test.ts` and `tests\collaborative\ot\ot_merged.test.ts`